source/read/change-streams.txt

```diff
@@ -0,0 +1,262 @@
+.. _pymongo-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your database. A change stream is a MongoDB Server feature that
+allows your application to subscribe to data changes on a collection, database,
+or deployment.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see
+:ref:`<pymongo-get-started>`.
+
+Open a Change Stream
+--------------------
+
+To open a change stream, call the ``watch()`` method. The instance on which you
+call the ``watch()`` method on determines the scope of events that the change
+stream listens for. You can call the ``watch()`` method on the following
+classes:
+
+- ``MongoClient``: To monitor all changes in the MongoDB deployment
+- ``Database``: To monitor changes in all collections in the database
+- ``Collection``: To monitor changes in the collection
+
+The following example opens a change stream on the ``restaurants`` collection
+and outputs changes to the shell as they occur:
+
+.. literalinclude:: /includes/read/change-streams.py
+   :start-after: start-open-change-stream
+   :end-before: end-open-change-stream
+   :language: python
+
+To begin watching for changes, run the application. Then, in a separate
+application or shell, modify the ``restaurants`` collection. The following
+example updates a document with a ``name`` field value of ``Blarney Castle``:
+
+.. _pymongo-change-stream-update:
+
+.. literalinclude:: /includes/read/change-streams.py
+   :start-after: start-update-for-change-stream
+   :end-before: end-update-for-change-stream
+   :language: python
+
+When you update the collection, the change stream application prints the change
+to the shell as it occurs. The change event printed to the shell resembles the
```

Comment:
"printed to the shell" sounds confusing for a python user.  I'd say something more like "the change event printed to the output".

Reply:
Simplified to just say "prints the change as it occurs"